docker/Dockerfile

```diff
@@ -30,3 +36,7 @@ RUN chmod a+x oommf
 
 # Mounting directory.
 WORKDIR /io
+
+# Quick sanity check:
+RUN oommf +version
+RUN oommf +platform
```

Comment:
When/where would the output of these commands (or a possible failure) be visible?

Reply:
The github workflow should show them.
If the error code is not zero, the building will be aborted and reported as an error.